src/_tutorials/language/futures.md

```diff
@@ -456,9 +470,11 @@ Nested callbacks also work, but they're harder to read and not as Dart-y.
 If the order of execution of the functions is not important, you can use
 `Future.wait()`.
 
-The functions get triggered in quick succession; when all of them complete
-with a value, `Future.wait()` returns a new Future.  This Future completes
-with a list containing the values produced by each function.
+When you pass `Future.wait()` a list of uncompleted futures,
```

Comment:
Technically, you can give it completed futures too. I would just say "a list of futures".